lib/qmp.ml

```diff
@@ -180,6 +218,9 @@ let message_of_string str =
     | "xen-load-devices-state"   -> json |> xen_load_devices_state   |> fun x -> Xen_load_devices_state x
     | "xen-set-global-dirty-log" -> json |> xen_set_global_dirty_log |> fun x -> Xen_set_global_dirty_log x
     | "blockdev-change-medium"   -> json |> blockdev_change_medium   |> fun (x, y) -> Blockdev_change_medium (x, y)
+    | "device_del"               -> json |> device_del               |> fun x -> Device_del x
+    | "device_add"               -> json |> device_add               |> fun (x, y, z) -> Device_add (x, y, z)
```

Comment:
Please swap the position of `del` and `add`

Reply:
updated.